docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension on Visual Studio Code
+
+You can install the {ProductShortName} extension on Visual Studio Code (VS Code) on Linux, Windows, or macOS.
+
+.Prerequisites
+
+* link:https://code.visualstudio.com[VS Code installed]
+* 8 GB RAM
+* JDK 11 or later
+* Environmental variable `JAVA_HOME` set to `jdk11`
+
+.Procedure
+
+. In VS Code, click the *Extensions* icon on the Activity bar.
+. Enter `{ProductName}` in the Search field.
+. Select the *Migration Toolkit for Applications* extension and click *Install*.
++
+The {ProductShortName} extension icon (image:vs_MTA_extension_icon.png[MTA code extension]) is displayed on the Activity bar.
+
+[NOTE]
+====
+To uninstall the {ProductShortName} extension, click the *Extensions* icon on the Activity bar, click the Manage icon of the {ProductShortName} extension, and select *Uninstall*.
```

Comment:
You have Extensions icon marked in bold and Manage icon not marked. Is there a tooltip or other indication that "Extensions" icon is called "Extensions"? If not, it should not be bold face.

Reply:
I just checked. Both icons have tooltips, so "Manage" should also be in bold.

Reply:
Suggest rewrite of note:
====
To uninstall the ... extension:
. In the *Extensions* view, select the *Migration Toolkit for Applications* extension.
. Click the *Manage* icon and select *Uninstall*.
====

You do not need to say "click the Extensions icon" if you specify Extensions view.
You do need to select the extension in order to access the Manage icon.
You need to spell out the name in full if you are referring to it in the UI because that is how it appears.
====